.github/workflows/release.yml

```diff
@@ -8,8 +8,6 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - ".github/workflows/release.yml"
```

Comment:
Isn't this going to be pushing a new image on every PR now? Or is the `push: ${{ github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/') }}` on line 56 preventing that?

Reply:
yeah that should be only true when it's pushing to main.

* [push to main](https://github.com/aws/s2n-quic/actions/runs/4983914226/jobs/8921555068#step:6:978)
* [this PR](https://github.com/aws/s2n-quic/actions/runs/4984037816/jobs/8921845475?pr=1763#step:6:959)